_dasldemos/tidy-data.md

```diff
@@ -4,7 +4,9 @@ last_modified_at: 2019-11-06
 main_author:
 primary_reviewers: ptvan
 ---
-All collaborations, even when your collaborator is your future self, benefit from all parties learning the art of tidying data.  In many cases the best time to do this is to implement [Tidy Data](https://vita.had.co.nz/papers/tidy-data.html) approaches when the data are generated in the first place, thus minimizing issues with data loss, corruption and loss of effort to cleaning messy data.
+
+Learning how to organize your data will help reduce errors, make analysis easier, and improve collaboration (even when your collaborator is your future self). A helpful theory of data organization is called [Tidy Data](https://vita.had.co.nz/papers/tidy-data.html), as outlined in a manuscript by Hadley Wickham.
+In many cases the best time implement tidy data is when the data are generated in the first place, thus minimizing issues with data loss, corruption and loss of effort to cleaning messy data.
```

Comment:
...best time **to** implement tidy data...

Reply:
thanks, fixed